lxd/storage/backend_lxd.go

```diff
@@ -2632,7 +2632,7 @@ func (b *lxdBackend) SetInstanceQuota(inst instance.Instance, size string, vmSta
 		// this will also pass empty quota for the config filesystem volume as well, allowing a former
 		// quota to be removed from both volumes.
 		if vmStateSize == "" && size != "" {
-			vmStateSize = deviceConfig.DefaultVMBlockFilesystemSize
+			vmStateSize = b.driver.Info().DefaultVMBlockFilesystemSize
```

Comment:
@roosterfish please could you land this change and the associated driver changes as a separate PR to keep this overall PR size down. Cheers

Reply:
Moved to another PR with https://github.com/canonical/lxd/pull/12566

Reply:
Moved with https://github.com/canonical/lxd/pull/12566